buildenv/jenkins/openjdk_tests

```diff
@@ -267,7 +267,13 @@ timestamps{
             env.EXTRA_OPTIONS = params.EXTRA_OPTIONS ? params.EXTRA_OPTIONS : ""
             env.BUILD_LIST = params.BUILD_LIST ? params.BUILD_LIST : ""
             SPEC = PLATFORM_MAP[params.PLATFORM]["SPEC"]
-            dockerAgents = PLATFORM_MAP[params.PLATFORM]["DockerAgents"] ? PLATFORM_MAP[params.PLATFORM]["DockerAgents"] : []
+            // Block dockerAgents on adoptium CI for riscv64 until we make it work
+            // This will allow RISC-V test jobs to queue up and not hang
+            if ( env.JENKINS_URL.contains("ci.adoptium.net")) {
```

Comment:
This will stop other linux test jobs using dynamic agents. Is this expected?

Reply:
For RISC-V specifically, maybe limit to `if ( env.JENKINS_URL.contains("ci.adoptium.net") && SPEC.equals('linux_riscv64'))` ?

Reply:
What other platforms do we do this for? I was under the impression it was only RISC-V